packages/frontend/src/fetchers/footer.ts

```diff
@@ -0,0 +1,56 @@
+import useSWR, { type SWRConfiguration } from 'swr'
+import type { FooterData } from '@/types/footer'
+
+const fetchFooter = async (): Promise<FooterData | null> => {
```

Comment:
[nice to have] could we add a `GET` endponit for footer query for better caching